src/main/java/app/persistence/MaterialMapper.java

```diff
@@ -13,11 +15,58 @@ public List<Material> getAllMaterials(String description, ConnectionPool connect
         return null;
     }
 
-    public void createMaterial(Material material, ConnectionPool connectionPool) {
+    public void createMaterial(Material material, ConnectionPool connectionPool) throws DatabaseException {
+        String sql = "INSERT INTO materials (material_id, material_description, material_unit, material_price) VALUES (?, ?, ?, ?)";
```

Comment:
The material id cannot be inserted at this time as it does not exist in the database yet and so cannot be retrieved/obtained. The id will be automatically generated and should not be set manually.